packages/node/src/NodeFileSystem.ts

```diff
@@ -69,4 +72,10 @@ export class NodeFileSystem implements FileSystem {
         })
     })
   }
+
+  public async loadPostgresPlugin(storageConfig: WalletStorageConfig, storageCreds: WalletStorageCreds) {
+    await storagePlugin.postgresstorage_init()
```

Comment:
I don't believe this is asynchronous. Could be wrong, but by default `ffi-napi` is sync.